app/views/nsm/steps/disbursement_cost/edit.html.erb

```diff
@@ -7,19 +7,20 @@
     <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
     <%= step_form @form_object do |f| %>
       <% if @form_object.other_disbursement_type? %>
-        <%= f.govuk_number_field :total_cost_without_vat, min: 1, step: 0.01, width: 5, label: { tag: 'strong' }, data: { selector: 'valueField', multipler: @form_object.multiplier } %>
```

Comment:
Also, you want to add `value: gbp_field_value(@form_object.total_cost_without_vat)` to this. That way if a user has entered e.g. "9.50", it won't prepopulate the field as "9.5"

Reply:
true and on both accounts and updated!